doc/2/api/controllers/security/delete-profile/index.md

```diff
@@ -42,6 +45,7 @@ Method: DELETE
 ### Optional:
 
 - `refresh`: if set to `wait_for`, Kuzzle will not respond until the profile deletion is indexed (default: `"wait_for"`)
+- `onAssignedUsers`: if set to `remove`, Kuzzle will remove this profile from assigned users. If this was a user's sole profile, then the `Annonymous` profile will be assigned. (default: `"fail"`) <SinceBadge version="auto-version" />
```

Comment:
Shouldn't we apply the `default` profile instead?

btw:

```suggestion
- `onAssignedUsers`: if set to `remove`, Kuzzle will remove this profile from assigned users. If this was a user's sole profile, then the `anonymous` profile will be assigned. (default: `"fail"`) <SinceBadge version="auto-version" />
```

Reply:
In some situations it could be better to have the `default` instead ? @Aschen

Reply:
I'm not comfortable with the `default` profile since this profile could have more right than the previous user profile.  
At least with the anonymous user we are 100% sure that we are not gonna have an unwanted privilege escalation